nvflare/app_common/logging/log_sender.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import threading
+
+from nvflare.apis.event_type import EventType
+from nvflare.apis.fl_constant import FLContextKey, ProcessType, StreamCtxKey
+from nvflare.apis.fl_context import FLContext
+from nvflare.apis.storage import DataTypes
+from nvflare.apis.workspace import Workspace
+from nvflare.app_common.logging.constants import LOG_STREAM_EVENT_TYPE, Channels
+from nvflare.app_common.streamers.file_streamer import FileStreamer
+from nvflare.widgets.widget import Widget
+
+
+class LogSender(Widget):
```

Comment:
Please change to ErrorLogSender since it only works for error logs.
You could refactor this later to make some part of the code reusable for other type of logging.